fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/iterators/RdfAdder.java

```diff
@@ -53,7 +56,38 @@
      */
     public RdfAdder(final IdentifierConverter<Resource, FedoraResource> idTranslator, final Session session,
```

Comment:
OK, I've removed that constructor and updated the unit test to use the new constructor instead.